src/models.rs

```diff
@@ -736,3 +736,30 @@ pub struct InstallationToken {
     pub permissions: Permissions,
     pub repositories: Option<Vec<Repository>>,
 }
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Why camelCase? I'm pretty sure GitHub uses "snake_case" in their JSON.

Reply:
Mhh you're right. Another example of missed copy-pasting :eyes: